decidim-core/lib/decidim/form_builder.rb

```diff
@@ -179,7 +179,7 @@ def editor(name, options = {})
       options[:lines] ||= 10
       options[:disabled] ||= false
 
-      content_tag(:div, class: "editor #{"hashtags__container" if options[:hashtaggable]}") do
+      content_tag(:div, class: "editor mentions__container #{"hashtags__container" if options[:hashtaggable]}") do
```

Comment:
wont' these changes affect hashtags?

Reply:
> I've left some questions.
> Take into account that the tests are not succeeding right now.

Yes, I'm on it (the tests).

About your questions: 
Actually I wanted to discuss the possibility of allowing mentions on proposals creation (feedback from product would be nice). Partly because I had no way to test otherwise in a normal textarea. In order to make it work all the added classes are needed.
But if we decide not to allow mentions, I'll withdraw the specific commit with that feature.

`mentions__container` and `hastags__container` are exactly the same currently, I don't thing that will affect the mixing:
decidim-core/app/assets/stylesheets/decidim/modules/_input-hashtags.scss
decidim-core/app/assets/stylesheets/decidim/modules/_input-mentions.scss

Reply:
removed now